frontend/src/utils/env.ts

```diff
@@ -0,0 +1,3 @@
+export function getGoogleAnalyticsAppId(){
+  return process.env.GA_ID || 'Unknown App ID'
```

Comment:
Do we need this?

Reply:
Probably not, no. I'll remove it and add an error log.